test/datahike/test/api.cljc

```diff
@@ -672,3 +672,115 @@
                   hash-2 (hash @conn)]
               (is (not= hash-1 hash-2))
               (is (not= hash-0 hash-2)))))))))
+
+(defn test-write-schema [id cfg]
```

Comment:
These tests are not related to directly testing the introduction of `tx-log`, right? They are here to test that what existed before is not broken, right?